internal/services/cosmos/cosmosdb_postgresql_cluster_resource.go

```diff
@@ -304,6 +310,22 @@ func (r CosmosDbPostgreSQLClusterResource) Attributes() map[string]*pluginsdk.Sc
 			Type:     pluginsdk.TypeString,
 			Computed: true,
 		},
+		"server_names": {
```

Comment:
We can shorten this to `servers`, `_names` is implicit and one of the attributes in the block is already called `name`
```suggestion
		"servers": {
```